lib/fastlane/plugin/fueled/actions/tag.rb

```diff
@@ -12,8 +12,7 @@ def self.run(params)
         Actions.lane_context[SharedValues::TAG_NAME] = tag_name
         if other_action.is_ci
           UI.message("Tagging #{tag_name}...")
-          other_action.add_git_tag(tag: tag_name)
-          other_action.push_git_tags(tag: tag_name)
+          other_action.sh("git tag -f \"#{tag_name}\" && git push origin \"refs/tags/#{tag_name}\" --force")
```

Comment:
Isn't better to split these two commands into two?
I remember that I introduced a failed build once when I entered an invalid tag name.

Reply:
I think it's fine having it like this. Wondering how could a tag name be invalid 🤔 ? Moreover, the `-f` option guarantees that the command won't fail even if an exact same tag already exists. Lastly, this has been the exact same command we had in the custom Bitrise steps, so I do think it's been bulletproofed over the last 3 years.

Reply:
It can happen in case the tag name contains whitespace.

<img width="399" alt="Screen Shot 2021-12-03 at 8 48 12 PM" src="https://user-images.githubusercontent.com/19314956/144656700-9fdfb863-f73f-482d-bdf5-ad01f0fdf8b2.png">

But please ignore my comment if you are sure that `tag_name` is always in the valid format

Reply:
Interesting, I did not know whitespaces would result in an invalid tag name 👍 Actually, I found [this doc](https://git-scm.com/docs/git-check-ref-format) about the supported formats, quite interesting.